chat-server/server.js

```diff
@@ -25,6 +28,38 @@ app.get("/", (request, response) => {
   response.sendFile(__dirname + "/index.html");
 });
 
+app.get("/messages", (request, response) => {
+  response.json(messages);
+});
+app.get("/messages/:ID", (request, response) => {
+  const oneMess = request.params.ID;
+  console.log(oneMess);
+  const filterdMessage = messages.find((one) => {
+    console.log(one.id);
+    return one.id == oneMess;
+  });
+  response.json(filterdMessage);
+});
+app.post("/messages", (req, res) => {
+  const bod = req.body;
+  const newMessage = {
+    id: messages.length,
+    from: bod.from,
```

Comment:
One of the requirements in the project is to validate that `from` and `text` are set - what will happen in your server if they're not set?